database/seed.js

```diff
@@ -29,19 +28,19 @@ const createData = async function() {
           what_you_did_not_like: faker.lorem.sentence(),
           fit: Math.floor(Math.random() * 7)
       };
-
-      let query = "INSERT INTO reviews SET ?";
-
-      await connection.query(query, review, (err, result) => {
-          if(err) {
-              console.log('Error loading data', err);
-          } else {
-              console.log('Successfully loaded data');
-          }
-      });
+      reviews.push(review);
     }
-
   };
+  let fields = Object.keys(reviews[0])
+  const opts = { fields };
+  jsonParser = new Parser(opts);
+  let csv = jsonParser.parse(reviews)
+  fs.writeFile('data.csv', csv, (err, result) => {
```

Comment:
If you change your schema at all to have hierarchical data it might be better to create a json file since csv has a hard time with that. The file will be larger and take a little longer to import though.  Json is also easier to read if you ever end up needing to inspect your data file

Reply:
Ill have to get get some guidance from you about creating a json file.  My next step is definitely to add hierarchical structure because my query times are super slow.

I ended up using the seeddb script to seed my database instead of this one for now though